CONTRIBUTING.md

```diff
@@ -70,6 +70,11 @@ $ uv pip install -e .
 $ source .venv/bin/activate
 ```
 
+Note that you can modify the existing `.env.dev` dotenv file for running client SDK tests via the following:
```

Comment:
I forgot one thing. The typical way to do these is that you have a template which is checked in and tracked by git but then you tell users to copy that template and modify it as per their needs. And you gitignore _that_ copy instead. Otherwise there is no way to update the template tomorrow.

Reply:
Great suggestions. Updated